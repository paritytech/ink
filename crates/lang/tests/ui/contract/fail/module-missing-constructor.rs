use ink_lang as ink;

#[ink::contract]
mod contact {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
