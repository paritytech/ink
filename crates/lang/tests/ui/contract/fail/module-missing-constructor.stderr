error: missing ink! constructor
  --> tests/ui/contract/fail/module-missing-constructor.rs:4:1
   |
4  | / mod contact {
5  | |     #[ink(storage)]
6  | |     pub struct Contract {}
7  | |
...  |
11 | |     }
12 | | }
   | |_^
